src/main/java/com/greenfoxacademy/responses/BadRequestResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.greenfoxacademy.responses;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+/**
+ * Created by BSoptei on 2/1/2017.
+ */
+public class BadRequestResponse extends CustomServerResponse {
+
+    @Override
+    public ResponseEntity generateResponse() {
+        return new ResponseEntity<>("", createResponseHeaders(), HttpStatus.BAD_REQUEST);
+    }
+}
```

Comment:
ok, fixed it,
crud responses need fine tuning though.
but /login and /register are cool
going to push soon